<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>180</y>
      <width>301</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="runButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QListView" name="cookingListView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>601</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="filePathLineEdit">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>401</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="browseButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse Button</string>
    </property>
   </widget>
   <widget class="QPushButton" name="setTimeButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Time</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="timeLineEdit">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>50</y>
      <width>401</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="kitchenButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Number Of Kitchens </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="kitchenLineEdit">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>90</y>
      <width>401</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="cookButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Number Of Cookers</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="cookerLineEdit">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>130</y>
      <width>401</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>638</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
